### DISCUSSION TITLE
主线op 22.03版本EOL，是否有计划升级至最新版本

### DISCUSSION THREAD

**Discussion by breeze303** [istoreos/istoreos#1620] [Q&A]
目前个人觉得不应该升至23.05版本，而是升至未来的24.0版本，因为22对比23内核都是5.X，而目前主线op跳跃至6.6内核进行适配工作，在此想询问下是否有计划进行未来版本的升级工作
![image](https://github.com/istoreos/istoreos/assets/122870904/e56a5106-a8e5-4400-ac99-cf8c6ff136d6)

**Comment by jjm2473** [comment]
嗯，我也打算以后跳过23版本。22.03应该也支持到2025年吧，我看openwrt官方也没在4月发过22.03的EOL版本。

**Reply by breeze303** [reply]
> 嗯，我也打算以后跳过23版本。22.03应该也支持到2025年吧，我看openwrt官方也没在4月发过22.03的EOL版本。

确实，eol延期了

**Reply by Ta-152** [reply]
> 嗯，我也打算以后跳过23版本。22.03应该也支持到2025年吧，我看openwrt官方也没在4月发过22.03的EOL版本。

😭确定要跳过吗，我看5.15内核才开始带kmod-atlantic这个包，这个是AQC107网卡的驱动
24不知道要等到什么时候了，而且软件包不是要改成什么.apk吗

**Reply by jjm2473** [reply]
@Ta-152 23.05之前出过测试固件，你可以试试

**Reply by Ta-152** [reply]
> @Ta-152 23.05之前出过测试固件，你可以试试

感谢,找了一下,23.05测试固件最新的是2023-09-22这个版本嘛?

**Reply by jjm2473** [reply]
@Ta-152 对

**Comment by haobinnan** [comment]
坐等
